autodoc-admin-api/data.lua

```diff
@@ -0,0 +1,1160 @@
+return {
+
+--------------------------------------------------------------------------------
+-- Known files and entities, to compare what's coded to what's documented
+--
+-- We avoid automagic detection based on traversal of filesystem and modules,
+-- so that adding new info to the documentation becomes a conscious process.
+-- We traverse the filesystem and modules to cross-check that everything that
+-- is present in the code is either documented or consciously omitted from
+-- the docs (e.g. in the last stage of deprecation).
+--------------------------------------------------------------------------------
+
+  known = {
+    general_files = {
+      "kong/api/routes/kong.lua",
+    },
+    nodoc_files = {
+      "kong/api/routes/cache.lua", -- FIXME should we document this?
+    },
+    entities = {
+      "services",
+      "routes",
+      "consumers",
+      "plugins",
+      "certificates",
+      "snis",
+      "upstreams",
+      "targets",
+    },
+    nodoc_entities = {
+      "apis",
+    },
+  },
+
+--------------------------------------------------------------------------------
+-- General (non-entity) Admin API route files
+--------------------------------------------------------------------------------
+
+  intro = [[
+
+    # Kong Admin API
```

Comment:
The page title, and the only H1 on the page, is set by the front matter `title: ` tag https://github.com/Kong/docs.konghq.com/blame/master/app/0.14.x/admin-api.md#L2